dm/dm/config/subtask.go

```diff
@@ -143,6 +146,16 @@ func (db *DBConfig) AdjustWithTimeZone(timeZone string) {
 	db.Adjust()
 }
 
+// FetchTimeZoneSetting fetch target db global time_zone setting.
+// TODO: move this function from TiDB to tiflow.
```

Comment:
what does this todo mean?

Reply:
I should refer to dbutil.GetTimeZoneOffset and FormatTimeZoneOffset. Will update the comment later